requirements.txt

```diff
@@ -3,7 +3,7 @@ django_readonly_field
 jsonschema>=3.0
 libcomps~=0.1.15
 productmd>=1.25
-pulpcore>=3.12
+pulpcore>=3.13
```

Comment:
pulpcore 3.13 is not out yet, please require pulpcore>=3.13.0.dev for CI to pass.
It will be updated to 3.13 at release time.